### ISSUE TITLE
Requirement of Munin::Master::GraphOld perl modules by munin_stats plugin inhibits separate packaging

### ISSUE THREAD

**Issue by yuwata** [munin-monitoring/munin#745]
Since the commit https://github.com/munin-monitoring/munin/commit/ec3128b4be6511d589056b8c190eb7c414b52b0f, munin_stats plugin requires Munin::Master::GraphOld perl modules. This causes munin.rpm is required munin-node.rpm, and cannot install munin-node.rpm without unwanted munin.rpm.
```
$ rpm -q --requires munin-node | grep Master
perl(Munin::Master::GraphOld)
$ rpm -q --provides munin | grep GraphOld
perl(Munin::Master::GraphOld)
$ rpm -q munin
munin-2.0.27-1.fc24.noarch
```
See also, https://bugzilla.redhat.com/show_bug.cgi?id=1392279

**Comment by kenyon** [comment]
Maybe we should allow the user to decide on graphing munin-graph time via an environment variable. That way this plugin will not depend on code from Munin Master. What do you think of that approach @steveschnepp and @h01ger?

**Comment by kenyon** [comment]
For now I think we should at least revert the commit so that Red Hat packages can be fixed, so I'll merge #759.

**Comment by kenyon** [comment]
I created a new issue to enhance the plugin a different way, so closing this issue.
